llama_index/bridge/langchain.py

```diff
@@ -1,7 +1,7 @@
 import langchain
 
 # prompts
-from langchain import BasePromptTemplate, PromptTemplate
+from langchain.prompts import BasePromptTemplate, PromptTemplate
```

Comment:
I think we need to also bump the minimum langchain version in the setup.py

Reply:
Done @logan-markewich